webform_import.module

```diff
@@ -437,20 +453,30 @@ function _webform_import_import($form, $form_state, $file) {
         if ($components_list[$fields[$k]]['form_key'] === 'webform_ip_address') {
           $ipaddress = $cell_value;
         }
-        if ($k === $submitted_key) {
-          $submitted = strtotime($cell_value);
+        // Timestamps for webform
+        if ($components_list[$fields[$k]]['form_key'] === 'webform_time') {
+          // Modify saved date string format for strtotime to parse it
+          $submitted = strtotime(preg_replace('/ - /', ' ', $cell_value));
+        }
+        if ($components_list[$fields[$k]]['form_key'] === 'webform_completed_time') {
+          $completed = strtotime(preg_replace('/ - /', ' ', $cell_value));
+        }
+        // The next 3 lines can probably be removed because webform_submission_insert will update the modified time to current
+        if ($components_list[$fields[$k]]['form_key'] === 'webform_modified_time') {
+          $modified = strtotime(preg_replace('/ - /', ' ', $cell_value));
         }
       }
 
       $submission = (object) array(
         'nid' => $webform->nid,
         'uid' => $uid,
         'submitted' => $submitted,
-        'completed' => 0,
-        'modified' => REQUEST_TIME,
+        'completed' => $completed,
+        'modified' => $modified,
         'remote_addr' => $ipaddress,
         'is_draft' => FALSE,
-        'serial' => NULL,
+	// use the row counter as the serial number if it's missing (NULL); needs to be unique or will cause DB error
```

Comment:
Capitalize and end with a period.
`Use the row counter as the serial number if it's missing (NULL); It needs to be unique or will cause a database error.`